### PR TITLE
[ examiner ]  fix can't refresh page after Edit

### PR DESCRIPTION
**Pull request by kongtiaowang**
[ examiner ]  fix can't refresh page after Edit.
                      Header location doesn't work.
https://github.com/aces/Loris/issues/5735

### CODE REVIEW

**Comment by maltheism** [comment]
```
--------------------------------------------------------------------------------
2507FOUND 1 ERROR AFFECTING 1 LINE
2508--------------------------------------------------------------------------------
2509 275 | ERROR | [x] Equals sign not aligned with surrounding assignments;
2510     |       |     expected 5 spaces but found 1 space
2511--------------------------------------------------------------------------------
2512PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
2513--------------------------------------------------------------------------------
```

**Comment by maltheism** [comment]
travis failing on 7.2 but passing on 7.3. I'm going to restart it.

File: modules/examiner/php/editexaminer.class.inc [hunk 1]
```diff
@@ -270,11 +270,12 @@ class EditExaminer extends \NDB_Form
...
-
-        $baseurl = \NDB_Factory::singleton()->settings()->getBaseURL();
-        $url     = "{$baseurl}/examiner/editExaminer/?identifier="
-               . $examinerID;
+        $factory = \NDB_Factory::singleton();
+        $baseURL = $factory->settings()->getBaseURL();
+        $url     = $baseURL."/examiner/editExaminer/?identifier="
+                   . $examinerID;
         header("Location: " . $url, true, 303);
+        exit();
```
Comment (driusan) [inline modules/examiner/php/editexaminer.class.inc]: This should not be hard exiting and the rest of the code seems to only be changing the case of the variable.. what are you trying to accomplish with this PR?

Comment (kongtiaowang) [reply modules/examiner/php/editexaminer.class.inc]: header location don't work without exit() or die(). I am fixing the header function.

**Pull request merged**